src/Components/Loader.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import LoaderLogo from '../loader.svg';
+
+
+const Loader = ({text}) => {
+    return(
+        <div style={{
+            display:'flex',
+            alignItems:'center',
+            justifyContent:'center',
+            flexDirection:'column',
+            height:window.innerHeight
```

Comment:
Try to use css classes for styles. 
```suggestion
            height:window.innerHeight
```